chain/network/src/stats/metrics.rs

```diff
@@ -1,122 +1,15 @@
-use crate::network_protocol::Encoding;
-use near_metrics::{
-    exponential_buckets, try_create_histogram, try_create_histogram_vec, try_create_int_counter,
-    try_create_int_counter_vec, try_create_int_gauge, Histogram, HistogramVec, IntCounter,
-    IntCounterVec, IntGauge, IntGaugeVec,
-};
-use near_network_primitives::types::{PeerType, RoutedMessageBody};
+use near_network_primitives::types::RoutedMessageBody;
```

Comment:
I am trying to understand why there are still some metrics left in this file.  Can you explain it please?

Reply:
The remaining ones are updated in multiple locations (it's not really a best practice), and fixing that has to wait for a refactor.